calculator/main.py

```diff
@@ -13,8 +12,11 @@ def create_new_calculator(operations=None):
     :param operations: Dict with initial operations.
                        ie: {'sum': sum_function, ...}
     """
-    pass
-
+    if operations == None:
```

Comment:
A couple of things here, when checking for None the more pythonic way would be `if operations is None:`. Also instead of checking to see if it is None and then setting it to a default value it's better to just set that default in the first place. This can be accomplished by having `def create_new_calculator(operations={})` as your function definition on line 6.